src/pages/loading-waiting-states/spinner-overlays/index.tsx

```diff
@@ -0,0 +1,209 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import {
+    AppBar,
+    Button,
+    Card,
+    CardActions,
+    CardHeader,
+    CardContent,
+    Checkbox,
+    CircularProgress,
+    Divider,
+    FormControlLabel,
+    Hidden,
+    IconButton,
+    Toolbar,
+    Typography,
+} from '@material-ui/core';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import MenuIcon from '@material-ui/icons/Menu';
+import { useDispatch } from 'react-redux';
+import { TOGGLE_DRAWER } from '../../../redux/actions';
+import { eulaDetails } from './eulaText';
+
+const useStyles = makeStyles((theme: Theme) => ({
+    container: {
+        padding: `${theme.spacing(2)}px ${theme.spacing(2)}px`,
+        maxWidth: '450px',
```

Comment:
This looks fine, but I would just make sure that between Angular and React we are using the same width for the EULA cards.

Reply:
As per design width should be 450px which I missed in Angular . Shall I fix this width issue for angular-design-pattern repo? So both card width will match.

Reply:
You can log an issue for it on the Angular repo one and we can fix it in a bug squash

Reply:
Logged an issue https://github.com/pxblue/angular-design-patterns/issues/104